### PR TITLE
Polish interface

### PR DESCRIPTION
**Pull request by Duncid**
- Various improvements to the look and feel of chat, fine tuning of Tab, Item, Avatar.
- Adding a new BarHeader component to handle a unified Bar component:
  - It's incomplete, we need to discuss how to finish it in a good way. We can merge as is though.

### CODE REVIEW

**Comment by spolu** [comment]
Deferring review to @fontanierh

File: front/components/sparkle/navigation.tsx [hunk 1]
```diff
@@ -80,11 +81,15 @@ export const topNavigation = ({
...
       icon: TestTubeIcon,
       href: `/w/${owner.sId}/u/gens`,
-      sizing: "expand",
+      sizing: "hug",
       current: current === "lab",
     });
   }
-
+  {
+    nav.push({
```
Comment (fontanierh) [inline front/components/sparkle/navigation.tsx:89]: This is really kind of a hack, let me see how we can improve a bit

**Comment by Duncid** [comment]
@fontanierh changed the way to do separator.
better?

**Comment by fontanierh** [comment]
@Duncid  yes way better ! Thanks

**Review by fontanierh — APPROVED** [review]

**Pull request merged**